src/Exception/UnsupportedDatabaseTypeException.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+namespace Port\Doctrine\Exception;
+
+class UnsupportedDatabaseTypeException extends \Exception {}
```

Comment:
Missing newline. Also every exception should at least implement `Port\Exception` interface.